src/pages/Home/components/BenefitsSection/BenefitsSection.js

```diff
@@ -1,8 +1,27 @@
 import { Benefit } from "./Benefit/Benefit"
+import information from './Benefit/imgs/information.png'
+import trophy from './Benefit/imgs/trophy.png'
+import handshake from './Benefit/imgs/handshake-heart.png'
+
 
 export const BenefitsSection = () => {
-    return <Benefit src='http://www.bfmriya.org.ua/wp-content/uploads/2022/08/about-e1659449409427.png' 
-    alt='information' 
-    upperText='Відкритість'
-    lowerText='До допомоги всім українцям.'/>
+    return (
+        <div className="benefits-section">
```

Comment:
You can use `<section>` tag instead of `<div>`. Or `<article>`

Reply:
+